### PR TITLE
NEP: add 3 missing data NEPs rescued from 2011-2012

### LINKED ISSUE #11617: unwanted dependency on graphviz for NEPs doc build
This was introduced in NEP 13 (ufunc overrides), it should be removed again. Currently the doc build looks for a `dot` command-line executable, and this name is apparently not even os-independent (see Sphinx docs [here](http://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html#confval-graphviz_dot)).

Possible alternative to removal: there seem to be some packages on PyPI that implement simplified versions of the Graphviz dot language. Probably not worth it (easier to just commit a couple of pngs), but is an option.

Current end of doc build:
```
Warning, treated as error:
dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
make: *** [html] Error 1
```

### PR DESCRIPTION
**Pull request by rgommers**
As proposed on the mailing list. I changed formatting and replaced a few links to gists/branches with references to other NEPs. And added a *Context* sentence at the start of each NEP. No other content changes.

Also fixes gh-11617, so one can actually build the neps site locally without having to have graphviz installed.

### CODE REVIEW

**Comment by charris** [comment]
Thanks Ralf.

File: doc/neps/nep-0026-missing-data-summary.rst [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+NEP 26 — Summary of Missing Data NEPs and discussion
+====================================================
+
+:Author: Mark Wiebe <mwwiebe@gmail.com>, Nathaniel J. Smith <njs@pobox.com>
+:Status: Deferred
+:Type: Standards Track
+:Created: 2012-04-22
+
+*Context: this NEP was written as summary of the large number of discussions
+and proposals (NEP 12, NEP 24, NEP 25), regarding missing data functionality.*
```
Comment (eric-wieser) [inline doc/neps/nep-0026-missing-data-summary.rst]: Will these auto-link?

File: doc/neps/nep-0026-missing-data-summary.rst [hunk 2]
```diff
@@ -0,0 +1,727 @@
...
+One option is to just copy R, by implementing a mechanism whereby
+dtypes can arrange for certain bitpatterns to be given NA semantics.
+
+One option is to copy numpy.ma closely, but with a more optimized
+implementation. (Or to simply optimize the existing implementation.)
+
+One option is that described in the NEP_, for which an implementation
+of mask-based missing data exists. This system is roughly:
+
+.. _NEP: https://github.com/numpy/numpy/blob/master/doc/neps/nep-0012-missing-data.rst
```
Comment (eric-wieser) [inline doc/neps/nep-0026-missing-data-summary.rst]: Can we link to the published NEP here?

File: doc/neps/nep-0026-missing-data-summary.rst [hunk 3]
```diff
@@ -0,0 +1,727 @@
...
+  a bitpattern NA to an array which supports both requires accessing
+  the data by "peeking under the mask".
+
+Another option is that described in the alterNEP_, which is to implement
+bitpattern dtypes with NA semantics for the "statistical missing data"
+use case, and to also implement a totally independent API for masked
+arrays with ignore semantics and all mask manipulation done explicitly
+through a .mask attribute.
+
+.. _alterNEP: https://gist.github.com/njsmith/1056379
```
Comment (eric-wieser) [inline doc/neps/nep-0026-missing-data-summary.rst]: This became NEP24, maybe?

**Comment by rgommers** [comment]
Thanks @eric-wieser. Addressed in gh-12019.

There may be a few more things; I'm trying to do this with fairly minimal effort.

**Pull request merged**